src/claimApplication/claimantInfo/paymentInfo.js

```diff
@@ -7,8 +9,20 @@ export const identifyingContent = {
 };
 
 export function changes() {
+  addStyles();
   rearrangeTable();
   updateRadioButtons();
+  document.addEventListener('headerReady', setNewTitle);
+}
+
+function addStyles() {
+  const style = document.createElement('style');  
+  style.innerHTML = `
+    .usa-legend {
+      max-width: fit-content;
+    }
+  `;
+  document.head.appendChild(style);
```

Comment:
this little change is to fix a weird UI thing where the legend had a max-width that made it break line in the middle of the screen